app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    copy_command, old_file, new_file = command.split(" ")
```

Comment:
The task checklist specifies that you should ensure the input string contains exactly three elements separated by spaces, and 'cp' should be the first element. Consider adding a validation step to check this condition before proceeding with the file operations.